packages/components/src/InsightsLabel/labels.scss

```diff
@@ -42,13 +13,6 @@
         }
     }
 }
-
-.ins-c-label-1.pf-c-label {
-    background-color: var(--ins-c-label--blue-bg);
-    .pf-c-label__content {
-        color: var(--inc-c-label--content-blue);
-        .pf-c-label__icon {
-            color: var(--inc-c-label--icon-blue);
-        }
-    }
-}
+.ins-c-label-2.pf-c-label {.pf-c-label__content::before {
+    border: 1px solid var(--inc-c-label--icon-gold);
```

Comment:
I have a feeling that the colors from https://marvelapp.com/prototype/g70a1he/screen/84434999 are different. Could you please double check it?

Reply:
Good catch. I asked Marie to give me the exact # number of the color.

Reply:
Updated the colors according to the Marie's mockup